lua/lspconfig/haxe_language_server.lua

```diff
@@ -0,0 +1,61 @@
+local configs = require 'lspconfig/configs'
+local util = require 'lspconfig/util'
+
+configs.haxe_language_server = {
+  default_config = {
+    cmd = {"haxe-language-server"};
+    filetypes = {"haxe"};
+    root_dir = util.root_pattern("build.hxml");
+    settings = {
+      ["haxe.executable"] = "haxe";
+    };
+    init_options = {
+      displayArguments = {"build.hxml"};
```

Comment:
I see, so this is the necessary minimal displayArgument? based on 
````
For this LSP config to work, your Haxe compiler arguments must be in a
file at the root of your project called `build.hxml`. Theoretically
you could configure this by setting `init_options.displayArguments`,
but no other value seems to work at the moment.
```
I assumed this would get overwritten.

Reply:
Yep. Based on the docs for the LS, I think you should be able to call this file whatever you want and adjust the setting accordingly. However, I found that `build.hxml` (a common choice in the Haxe community) was the only thing that worked for me. 🤷🏻‍♂️

Reply:
@mjlbach This actually looks to be working as expected—not sure if the project I initially tested it with was set up incorrectly or what. I left the default value as `"build.hxml"` since that is quite common, but updated the docs to reflect that this can be any file you choose. How does that sound?

Reply:
Sounds good, I have no experience with haxe so I’m just armchair reviewing